diff --git a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/BitmapUtils.kt b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/BitmapUtils.kt
index 58a0e5b..b0da468 100644
--- a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/BitmapUtils.kt
+++ b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/BitmapUtils.kt
@@ -9,8 +9,8 @@ import android.graphics.YuvImage
 import android.media.Image.Plane
 import android.util.Log
 import com.mrousavy.camera.core.FrameInvalidError
-import com.mrousavy.camera.core.types.Orientation
-import com.mrousavy.camera.frameprocessors.Frame
+import com.mrousavy.camera.types.Orientation
+import com.mrousavy.camera.frameprocessor.Frame
 import java.io.ByteArrayOutputStream
 import java.nio.ByteBuffer
 
diff --git a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPackage.kt b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPackage.kt
index 2bd3ed6..0d0aa04 100644
--- a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPackage.kt
+++ b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPackage.kt
@@ -4,7 +4,7 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.bridge.ReactApplicationContext
 import com.facebook.react.uimanager.ViewManager
 import com.facebook.react.bridge.NativeModule
-import com.mrousavy.camera.frameprocessors.FrameProcessorPluginRegistry
+import com.mrousavy.camera.frameprocessor.FrameProcessorPluginRegistry
 
 class RNVisionCameraOCRPackage : ReactPackage {
     companion object {
diff --git a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPlugin.kt b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPlugin.kt
index 5ca4c32..00994f9 100644
--- a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPlugin.kt
+++ b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/RNVisionCameraOCRPlugin.kt
@@ -15,9 +15,9 @@ import com.google.mlkit.vision.text.devanagari.DevanagariTextRecognizerOptions
 import com.google.mlkit.vision.text.japanese.JapaneseTextRecognizerOptions
 import com.google.mlkit.vision.text.korean.KoreanTextRecognizerOptions
 import com.google.mlkit.vision.text.latin.TextRecognizerOptions
-import com.mrousavy.camera.frameprocessors.Frame
-import com.mrousavy.camera.frameprocessors.FrameProcessorPlugin
-import com.mrousavy.camera.frameprocessors.VisionCameraProxy
+import com.mrousavy.camera.frameprocessor.Frame
+import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin
+import com.mrousavy.camera.frameprocessor.VisionCameraProxy
 import android.graphics.Bitmap
 
 class RNVisionCameraOCRPlugin(proxy: VisionCameraProxy, options: Map<String, Any>?) :
diff --git a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/VisionCameraTranslatorPlugin.kt b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/VisionCameraTranslatorPlugin.kt
index 59491ac..4deaac9 100644
--- a/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/VisionCameraTranslatorPlugin.kt
+++ b/node_modules/react-native-vision-camera-ocr-plus/android/src/main/java/com/rnvisioncameraocr/VisionCameraTranslatorPlugin.kt
@@ -14,9 +14,9 @@ import com.google.mlkit.vision.common.InputImage
 import com.google.mlkit.vision.text.Text
 import com.google.mlkit.vision.text.TextRecognition
 import com.google.mlkit.vision.text.latin.TextRecognizerOptions
-import com.mrousavy.camera.frameprocessors.Frame
-import com.mrousavy.camera.frameprocessors.FrameProcessorPlugin
-import com.mrousavy.camera.frameprocessors.VisionCameraProxy
+import com.mrousavy.camera.frameprocessor.Frame
+import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin
+import com.mrousavy.camera.frameprocessor.VisionCameraProxy
 
 private const val TAG = "VisionCameraTranslator"
 
