diff --git a/node_modules/react-native-image-picker/android/build.gradle b/node_modules/react-native-image-picker/android/build.gradle
index 1234567..abcdefg 100644
--- a/node_modules/react-native-image-picker/android/build.gradle
+++ b/node_modules/react-native-image-picker/android/build.gradle
@@ -34,6 +34,10 @@ android {
             minifyEnabled false
         }
     }
+
+    buildFeatures {
+        buildConfig true
+    }
 
     sourceSets {
         main {
@@ -67,7 +71,7 @@ dependencies {
     if (isNewArchitectureEnabled() && getReactNativeMinorVersion() < 71) {
       implementation project(":ReactAndroid")
     } else {
-      implementation 'com.facebook.react:react-native:+' // from node_modules
+      compileOnly "com.facebook.react:react-android:0.73.6"
     }
 
     implementation "androidx.core:core:1.3.1"
