diff --git a/node_modules/react-native-pdf-lib/android/build.gradle b/node_modules/react-native-pdf-lib/android/build.gradle
index 0a3c611..fea3b69 100644
--- a/node_modules/react-native-pdf-lib/android/build.gradle
+++ b/node_modules/react-native-pdf-lib/android/build.gradle
@@ -14,6 +14,7 @@ apply plugin: 'com.android.library'
 android {
     compileSdkVersion 28
     buildToolsVersion '28.0.0'
+    namespace 'com.hopding.pdflib'
 
     dexOptions {
         jumboMode true
